diff --git a/node_modules/starlight-blog/components/Preview.astro b/node_modules/starlight-blog/components/Preview.astro
index 7b2a7e1..eeeacc8 100644
--- a/node_modules/starlight-blog/components/Preview.astro
+++ b/node_modules/starlight-blog/components/Preview.astro
@@ -18,9 +18,10 @@ const { entry, locale } = Astro.props
 
 <article class="preview">
   <header>
-    <h2>
+    <h2 class={entry.data.subtitle ? 'has-subtitle' : undefined}>
       <a href={getRelativeUrl(`/${getPathWithLocale(entry.id, locale)}`)}>{entry.data.title}</a>
     </h2>
+    {entry.data.subtitle && <p class="preview-subtitle">{entry.data.subtitle}</p>}
     {entry.data.cover && <Cover cover={entry.data.cover} />}
     <Metadata entry={entry} {locale} />
   </header>
@@ -43,7 +44,18 @@ const { entry, locale } = Astro.props
     margin-bottom: 1.5rem;
   }
 
+  h2.has-subtitle {
+    margin-bottom: 1rem;
+  }
+
   h2 a {
     text-decoration: none;
   }
+
+  .preview-subtitle {
+    margin-top: -1rem;
+    margin-bottom: 1.5rem;
+    font-size: var(--sl-text-lg);
+    color: var(--sl-color-gray-3);
+  }
 </style>
